src/thor/costmatrix.cc

```diff
@@ -294,8 +298,6 @@ void CostMatrix::Initialize(
     const google::protobuf::RepeatedPtrField<valhalla::Location>& source_locations,
     const google::protobuf::RepeatedPtrField<valhalla::Location>& target_locations) {
 
-  targets_->reserve(max_reserved_labels_count_);
```

Comment:
funnily enough it's better for performance if we don't reserve anything in the robin hood hash (3-4% improvement)